part-3/pk-sim-compounds-definition-and-work-flow.md

```diff
@@ -202,6 +202,8 @@ with ![Image](../assets/images/part-3/Fx-organ.png) = volume fraction of water,
 
 ![Image](../assets/images/part-3/fu-plasma.png) = free fraction in plasma.
 
+-------------
```

Comment:
Can we check if GitBook supports any kind of mathematical equations? Usually, in GitHub it is done like this:

`$\frac{x}{y}^2$`

$\frac{x}{y}^2$

Using graphics for formulae is horrible.